src/components/MapPage/MapPage.css

```diff
@@ -0,0 +1,7 @@
+#mapTitle {
```

Comment:
find a way to center using css properties that are specifically for that.
on my screen it is not centered
![image](https://github.com/user-attachments/assets/0cc08779-6b75-4d38-adf3-3f48fc36df21)